bufstream/docker-compose-quickstart/README.md

```diff
@@ -0,0 +1,23 @@
+![The Buf logo](https://raw.githubusercontent.com/bufbuild/buf-examples/main/.github/buf-logo.svg)
+
+# Docker Compose for Bufstream Quickstart
```

Comment:
I would still say remove "quickstart", and just call this directory `docker-compose`